virtualization-and-containers/overview.md

```diff
@@ -541,4 +541,4 @@ Source: [https://docs.docker.com/get-started/overview/](https://docs.docker.com/
 - ["Should I use Vagrant or Docker for creating an isolated environment?"](https://stackoverflow.com/questions/16647069/should-i-use-vagrant-or-docker-for-creating-an-isolated-environment)
 - [Malicious Docker Hub Container Images Used for Cryptocurrency Mining](https://www.trendmicro.com/vinfo/fr/security/news/virtualization-and-cloud/malicious-docker-hub-container-images-cryptocurrency-mining)
 - ["How To Make Package Managers Cry"](https://archive.fosdem.org/2018/schedule/event/how_to_make_package_managers_cry/)
-- [Open Container Initiative (OCI)](https://opencontainers.org/)
\ No newline at end of file
+- [Open Container Initiative (OCI)](https://opencontainers.org/)
```

Comment:
Is there a formatting change here?

Reply:
No, I didn't touch that line, but it still shows up in the commit.